src/app/visual/readonly-grid/readonly-grid-context-menu.component.html

```diff
@@ -0,0 +1,30 @@
+<sky-dropdown
+  buttonType="context-menu"
+>
+  <sky-dropdown-menu>
+    <sky-dropdown-item>
+      <button
```

Comment:
If I navigate to a dropdown with the keyboard controls, open the dropdown, then hit `tab`, the focus is placed on the next dropdown button. Is that expected?

Reply:
this was something we talked about in flex as not being the most ideal but something we could put up with. We still have to allow the tab key in popups/editors because a lot of sky components use tab to navigate them, like the datepicker. Without modifying each sky component I couldn't really find an easy way around it. Once they hit tab again or use the arrow keys it resumes the expected focus/highlight.

Reply:
Got it. Thanks for the explanation.